plugins/modules/user_role_workflow_manager.py

```diff
@@ -3152,9 +3203,14 @@ def update_user_role_profile_messages(self):
         self.result["changed"] = False
         result_msg_list = []
 
-        if self.created_user:
-            create_user_msg = "User(s) '{0}' created successfully in Cisco Catalyst Center.".format("', '".join(self.created_user))
-            result_msg_list.append(create_user_msg)
+        if self.want.get("password_update") is not True:
```

Comment:
self.created_user is common in both if and else. can we optimize?

```
if self.want.get("password_update") is not True:
    update_action = "created"
else:
    update_action = "updated"

if self.created_user:
    create_user_msg = "User(s) '{0}' {1} successfully in Cisco Catalyst Center.".format(
        "', '".join(self.created_user), update_action
    )
    result_msg_list.append(create_user_msg)
```

Reply:
Updated the code accordingly.